packages/sit-onyx/src/components/examples/ComponentShowcase/ComponentShowcase.vue

```diff
@@ -0,0 +1,367 @@
+<script lang="ts" setup>
+import arrowSmallRight from "@sit-onyx/icons/arrow-small-right.svg?raw";
+import checkSmall from "@sit-onyx/icons/check-small.svg?raw";
+import logout from "@sit-onyx/icons/logout.svg?raw";
+import settings from "@sit-onyx/icons/settings.svg?raw";
+import shareIos from "@sit-onyx/icons/share-ios.svg?raw";
+import { ref } from "vue";
+import {
+  OnyxAvatar,
+  OnyxAvatarStack,
+  OnyxBadge,
+  OnyxButton,
+  OnyxCheckboxGroup,
+  OnyxColorSchemeMenuItem,
+  OnyxIcon,
+  OnyxIconButton,
+  OnyxInput,
+  OnyxMenuItem,
+  OnyxNavBar,
+  OnyxNavButton,
+  OnyxNavItem,
+  OnyxPagination,
+  OnyxRadioGroup,
+  OnyxSelect,
+  OnyxStepper,
+  OnyxSwitch,
+  OnyxTable,
+  OnyxTag,
+  OnyxTextarea,
+  OnyxToastMessage,
+  OnyxUserMenu,
+  type SelectOption,
+} from "../../..";
+
+type State = {
+  select?: string[];
+  textarea?: string;
+  pagination: number;
+  checkboxGroup?: number[];
+  radioGroup?: number;
+  stepper?: number;
+};
+
+const isDark = defineModel<boolean>("dark");
+
+const selectOptions = Array.from({ length: 6 }, (_, index) => {
+  return {
+    label: `Option ${index + 1}`,
+    value: index + 1,
+  } satisfies SelectOption;
+});
+
+const checkboxGroupOptions = Array.from({ length: 3 }, (_, index) => {
+  return {
+    label: "Checkbox",
+    value: index + 1,
+  } satisfies SelectOption;
+});
+
+const radioGroupOptions = Array.from({ length: 4 }, (_, index) => {
```

Comment:
Another idea: Give the options different names, so it's more fun :)

Reply:
I thought of this too but in my opinion it looks cleaner / more minimal this way